### PR TITLE
Updating tutorial to match source

### PR DESCRIPTION
**Pull request by jimangel**
Most of the details are captured in this comment: https://github.com/kubernetes/examples/pull/410#issuecomment-862678482

A PR was opened to remove inappropriate terminology on k/website. Instead of being a text-only edit, the author chose to replace the tutorial with a new backend. At the time, we valued the contribution over bike shedding a backend. However, it recently came to my attention that SIG Docs is not the "true source" for the tutorial (details linked above). In addition, Google already modified the tutorials to remove inappropriate terminology.

The sample code ALSO exists in [k/examples/guestbook](https://github.com/kubernetes/examples/tree/master/guestbook) and THAT needs to be updated to match or be removed.

This PR brings https://kubernetes.io/docs/tutorials/stateless-application/guestbook/ to parity with https://cloud.google.com/kubernetes-engine/docs/tutorials/guestbook as well as reference the "source of truth."

There is one minor issue, the GCP frontend image is broke due to a PR changing the Dockerfile. This PR is contingent on https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/190 merging because that will cause a new working image to be published. I'm assuming this might have a unique tag too.

Next steps:
- [ ] Merge revert https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/190 in GCP org
- [ ]  Build the new container image (confirmed the merge will auto-build and the tag will remain the same)
- [ ] Merge this working PR into k/website
- [ ] Update [k/examples/guestbook](https://github.com/kubernetes/examples/tree/master/guestbook) to match or delete

Preview link: https://deploy-preview-28452--kubernetes-io-master-staging.netlify.app/docs/tutorials/stateless-application/guestbook/

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Hello @jimangel !

It seems that Netlify build failed.

Netlify Deploy log:
https://app.netlify.com/sites/kubernetes-io-master-staging/deploys/60cad6b85692b30008a2af17
> Error building site: "/opt/build/repo/content/en/docs/tutorials/stateless-application/guestbook.md:82:1": failed to render shortcode "codenew": failed to process shortcode: "/opt/build/repo/layouts/shortcodes/codenew.html:14:16": execute of template failed: template: shortcodes/codenew.html:14:16: executing "shortcodes/codenew.html" at <readFile $filename>: error calling readFile: stat /opt/build/repo/content/en/examples/application/guestbook/redis-service.yaml: no such file or directory

I think that including `redis-service.yaml` in this PR will solve the problem. 😊

File: content/en/examples/application/guestbook/frontend-deployment.yaml [hunk 1]
```diff
@@ -1,32 +1,29 @@
+# SOURCE: https://cloud.google.com/kubernetes-engine/docs/tutorials/guestbook
```
Comment (sftim) [inline content/en/examples/application/guestbook/frontend-deployment.yaml:1]: Nice

**Comment by sftim** [comment]
Approach LGTM

**Comment by jimangel** [comment]
@jihoon-seo thanks for the investigation! I just force pushed the change.

In this case, I missed renaming the `redis-service.yaml` to `redis-leader-service.yaml` (because now there's two services).

**Comment by jihoon-seo** [comment]
Direct links to the preview:
- https://deploy-preview-28452--kubernetes-io-master-staging.netlify.app/docs/tutorials/
- https://deploy-preview-28452--kubernetes-io-master-staging.netlify.app/docs/tutorials/stateless-application/guestbook/

/lgtm

**Comment by tengqm** [comment]
/approve

Needs a follow up to make sure the examples test is updated.

**Pull request merged**